webknossos/webknossos/skeleton/skeleton.py

```diff
@@ -47,12 +47,6 @@ class Skeleton(Group):
     _enforced_id: Optional[int] = attr.field(default=None, eq=False, repr=False)
     name: str = attr.field(default="Root", init=False, eq=False, repr=False)
 
-    """Should not be used with `Skeleton`, this attribute is only useful for sub-groups. Set to `Root`."""
```

Comment:
@jstriebel Do you know why the `_children` property was here redefined in the first place? I removed it now as it's redundant to `Group`, but it might exist for a reason? Maybe documenting that it should not be used for the `Skeleton` class? 

The comment `"""Should not be used with `Skeleton`, this attribute is only useful for sub-groups. Set to `Root`."""` probably belonged to `name`. Readding it now.

Reply:
> @jstriebel Do you know why the `_children` property was here redefined in the first place? I removed it now as it's redundant to `Group`, but it might exist for a reason? Maybe documenting that it should not be used for the `Skeleton` class?

I'm guessing that it might have been to define a certain order for the repr, but not quite sure :shrug: 

> The comment `"""Should not be used with `Skeleton`, this attribute is only useful for sub-groups. Set to `Root`."""` probably belonged to `name`. Readding it now.

Yes, the attribute-docstrings unfortunately are under their attribute :+1: